typings/single-spa.d.ts

```diff
@@ -105,8 +105,13 @@ declare module "single-spa" {
   export function triggerAppChange(): Promise<any>;
 
   // './applications/app-errors.js'
-  export function addErrorHandler(handler: (error: Error) => void): void;
-  export function removeErrorHandler(handler: (error: Error) => void): void;
+  type AppError = Error & {
+    appOrParcelName: string;
+    message: string;
```

Comment:
Error objects already have a `message` property - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message.

As such, I think we can remove this from our definition and rely on `Error` instead to have it.

Reply:
Ah yes thanks I'll make that change.

Perhaps the docs should be updated as well to reflect this? Or perhaps the docs are highlighting the `message` property as useful in debugging?

Reply:
> perhaps the docs are highlighting the message property as useful in debugging?

Yep exactly

Reply:
:+1: